### PR TITLE
Remove unused config param

### PR DESCRIPTION
**Pull request by seelabs**
@rec @scottschurr Pretty trivial changes. Shouldn't require much review.

### CODE REVIEW

**Comment by scottschurr** [comment]
What do we know about the npm failures?

File: src/ripple/app/tx/impl/CreateOffer.cpp [hunk 1]
```diff
@@ -239,13 +239,13 @@ CreateOffer::bridged_cross (
         throw std::logic_error ("Bridging with XRP and an endpoint.");
 
     OfferStream offers_direct (view, view_cancel,
-        Book (taker.issue_in (), taker.issue_out ()), when, ctx_.config, j_);
+        Book (taker.issue_in (), taker.issue_out ()), when, j_);
```
Comment (rec) [inline src/ripple/app/tx/impl/CreateOffer.cpp]: `j_`, hmm?  :-)  (Not an action item...)

**Comment by rec** [comment]
This change is obviously correct.  

I'm going to ignore the npm failures, as "endemic" and "obviously not generated by this code."

:+1:

**Comment by rec** [comment]
(Of course, we should fix the npm failures and our CI in general, but we shouldn't muddy this tiny change with that big problem...)

**Comment by scottschurr** [comment]
The npm failures are clearly our usual continuous integration noise.  Nice change.  Amazing how pervasive that unused parameter was.  :+1:

**Comment by seelabs** [comment]
@scottschurr The npm tests pass locally. Not sure what to do about the AppVeyor fail, but I'm comfortable that it's unrelated to these changes (it seems to have failed on `npm install`). At any rate, thanks for the thumbs up.

File: src/ripple/app/tx/impl/CreateOffer.cpp (hunk 1 above)
Comment (vinniefalco) [inline src/ripple/app/tx/impl/CreateOffer.cpp]: What about `j_`?

Comment (rec) [inline src/ripple/app/tx/impl/CreateOffer.cpp]: Sorry, it's just a funny name for a member variable!

Comment (vinniefalco) [inline src/ripple/app/tx/impl/CreateOffer.cpp]: We use `j` and `j_` to mean the relevant instance of `beast::Journal` since it occurs so frequently

**Comment by scottschurr** [comment]
Incorporated as d015debe2b76deaa1bf0deb6d8a53de33ba71aab.

**Pull request closed without merging**